src/fortran/client.F90

```diff
@@ -173,13 +186,15 @@ logical function key_exists(self, key)
   c_key = trim(key)
   c_key_length = len_trim(key)
 
-  key_exists = key_exists_c( self%client_ptr, c_key, c_key_length)
+  key_exists = key_exists_c(self%client_ptr, c_key, c_key_length, exists)
```

Comment:
So this is the alternate way of doing the result that was mentioned previously. I'm ok with one way or the other, it's more important to just be consistent.

Reply:
The entire API now returns error codes now, so I think it's consistent. The former boolean response of the `*_exists()` and `poll_*()` routines is now an additional 'exists' parameter at the end of those routines. It's perhaps confusing that `key_exists` and `exists` are so lexically close; this makes adopting your previous suggestion of putting in a dummy variable for function return codes extra useful.